grails-app/utils/org/broadinstitute/orsp/utils/UtilityClass.groovy

```diff
@@ -150,6 +152,23 @@ class UtilityClass {
         }
     }
 
+    void registerIssueListMarshaller() {
+        JSON.createNamedConfig(ISSUE_LIST) {
+            SimpleDateFormat  sd = new SimpleDateFormat("yyyy-MM-dd")
+            it.registerObjectMarshaller( Issue ) { Issue ie ->
```

Comment:
Standard convention is to just use `it ->`. In this case, `ie` doesn't seem to be an abbreviation for anything recognizable.